src/libraries/System.Private.CoreLib/src/System/Environment.cs

```diff
@@ -201,21 +201,17 @@ public static OperatingSystem OSVersion
             }
         }
 
+        internal const string InternalVersion = "9.0.0";
```

Comment:
I do not think we want to add yet another place to update for version bumps. If you want to do this, it needs to be wired to eng/Versions.props.

Reply:
The version update bump looks like this https://github.com/dotnet/runtime/commit/96c2e1d099a427a0c7f432c0c1ff7b2ec485b583 once a year. Is it really a big deal to update a string? If you think it is, lets close this PR then. It was a simple improvement with no visible effect on production bits.

Reply:
The version bumps are done every month in servicing, and they look like this: https://github.com/dotnet/runtime/pull/101779/

Reply:
Ah, the patch version. 🤭

Let me think a bit how to dedup it.